src/models/create_access_token_v2_request.rs

```diff
@@ -0,0 +1,28 @@
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+
+pub struct CreateAccessTokenV2Request {
+    #[serde(rename = "user_id")]
+    pub user_id: String,
+    #[serde(rename = "duration_in_minutes")]
+    pub duration_in_minutes: u64,
+
+    #[serde(
+        rename = "with_active_org_support",
+        skip_serializing_if = "Option::is_none"
+    )]
+    pub with_active_org_support: Option<bool>,
```

Comment:
same comment here to the go library - want to derive this field from whether active_org_id is some?